### DISCUSSION TITLE
Why can’t webp still images be displayed correctly?

### DISCUSSION THREAD

**Discussion by sasatadamoto** [wezterm/wezterm#6236] [Q&A]
### What Operating System(s) are you running on?

Windows

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

I tried displaying images on wezterm. Using the imgcat command, I was able to display jpg and png files, but webp still images cannot be displayed (to be specific, the area where the image should be displayed is filled with black pixels). 

Despite this, webp animated images can be displayed! Is this a specification of imgcat, or is it a bug on the wezterm side?(By the way, this phenomenon was also reproduced on Linux (Rocky Linux 9).)

**Comment by nik-rev** [comment]
Yeah. 
![image](https://github.com/user-attachments/assets/fe2d52a6-52e0-44f0-bffd-79dc4deffbd9)

webp images are black. As an emerging standard, webps have largely replaces pngs in almost all cases. So it's unfortunate they don't display properly

Interestingly enough, wezterm *has* the capability to display them. In yazi you can see them

![image](https://github.com/user-attachments/assets/b304bcb8-0523-4a05-bd54-30897b235140)
